tests/test_metrics.py

```diff
@@ -0,0 +1,44 @@
+import torch
+import pytest
+import numpy as np
+
+from brats.metrics import dice_score_one_class
+
+BATCH_DIMS = (2, 1, 5, 5, 5)
+CHANNEL_DIM = 1
+
+
+class TestDiceScoreOneClass:
+    def test_if_returns_1_for_perfect_fit(self):
+        images = torch.ones(*BATCH_DIMS)
+        target = torch.ones(*BATCH_DIMS)
+        assert np.isclose(dice_score_one_class(images, target), 1, atol=1.e-4)
+
+    def test_if_returns_0_for_worst_fit(self):
+        images = torch.zeros(*BATCH_DIMS)
+        target = torch.ones(*BATCH_DIMS)
+        assert np.isclose(dice_score_one_class(images, target), 0, atol=1.e-4)
+
+    def test_if_works_for_non_binary_data(self):
```

Comment:
Name is misleading. It should say test_assertion_error_for_non_binary_data or something like that.

Reply:
Removed this test as we agreed that the input does not to be binary